routes/userRoutes.js

```diff
@@ -37,6 +37,23 @@ router.delete('/:id', requireLogin, async (req, res) => {
   }
 });
 
+// GET User by ID
+router.get('/:id', requireLogin, async (req, res) => {
```

Comment:
Be careful with this route, as I encountered an error in studentroutes where it would prioritise a route "/:id" over "/:id/subscription" for example, so if you would maybe make a future route called "/:id/create" there could be an error

Reply:
I think the `:id` should always be the last part of the URL

Reply:
Yes that is also the way to avoid it